<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editProfessor</class>
 <widget class="QWidget" name="editProfessor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(6, 17, 60);
color: rgb(255, 255, 255);
font: 13pt &quot;Optima&quot;;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_Fname">
       <property name="styleSheet">
        <string notr="true">COLOR:white;
font: 15pt &quot;Optima&quot;;</string>
       </property>
       <property name="text">
        <string>First Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Fname">
       <property name="styleSheet">
        <string notr="true">BACKGROUND:rgb(255, 255, 255);
color: rgb(0, 0, 0);
	

	border-style: solid;
	border-width: 0px;
	border-radius: 16px;

	padding: 8px;</string>
       </property>
       <property name="maxLength">
        <number>20</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Sname">
       <property name="styleSheet">
        <string notr="true">COLOR:white;
font: 15pt &quot;Optima&quot;;</string>
       </property>
       <property name="text">
        <string>Last name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Sname">
       <property name="styleSheet">
        <string notr="true">BACKGROUND:rgb(255, 255, 255);
	color:rgb(0, 0, 0);


	border-style: solid;
	border-width: 0px;
	border-radius: 16px;

	padding: 8px;

</string>
       </property>
       <property name="maxLength">
        <number>20</number>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_age">
       <property name="styleSheet">
        <string notr="true">COLOR:white;
font: 15pt &quot;Optima&quot;;</string>
       </property>
       <property name="text">
        <string>Age</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_age">
       <property name="styleSheet">
        <string notr="true">BACKGROUND:rgb(255, 255, 255);


	
color: rgb(0, 0, 0);

	border-style: solid;
	border-width: 0px;
	border-radius: 16px;

	padding: 8px;
</string>
       </property>
       <property name="maxLength">
        <number>2</number>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_email">
       <property name="styleSheet">
        <string notr="true">COLOR:white;
font: 15pt &quot;Optima&quot;;</string>
       </property>
       <property name="text">
        <string>E-mail</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_email">
       <property name="styleSheet">
        <string notr="true">BACKGROUND:rgb(255, 255, 255);
color: rgb(0, 0, 0);

	

	border-style: solid;
	border-width: 0px;
	border-radius: 16px;

	padding: 8px;
</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_phone_number">
       <property name="styleSheet">
        <string notr="true">COLOR:white;
font: 15pt &quot;Optima&quot;;</string>
       </property>
       <property name="text">
        <string>phone number</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_phone">
       <property name="styleSheet">
        <string notr="true">BACKGROUND:rgb(255, 255, 255);
color: rgb(0, 0, 0);


	border-style: solid;
	border-width: 0px;
	border-radius: 16px;

	padding: 8px;</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_title">
       <property name="font">
        <font>
         <family>Optima</family>
         <pointsize>15</pointsize>
         <italic>false</italic>
         <bold>false</bold>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">COLOR:white;
font: 15pt &quot;Optima&quot;;</string>
       </property>
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_title">
       <property name="styleSheet">
        <string notr="true">BACKGROUND:rgb(255, 255, 255);
color: rgb(0, 0, 0);

	

	border-style: solid;
	border-width: 0px;
	border-radius: 16px;

	padding: 8px;</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhLatinOnly</set>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>50</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_edit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 13pt &quot;Optima&quot;;
background-color: #FF8C32;
	color: #fff;
	font-weight: bold;
	border-style: solid;
	border-width: 1px;
	border-radius: 17px;
	border-color: #f0742f;
	padding: 8px;

</string>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
